### ISSUE TITLE
Fullscreen edit mode closes on edit

### ISSUE THREAD

**Issue by enriquemorenotent** [sanity-io/sanity#1061]
I have set up a certain document to have an array of objects/documents, and those objects contain a "block" type.

If I try to edit those objects and open the fullscreen editor, it closes in the moment I click on the text area

![peek 2018-11-24 21-10](https://user-images.githubusercontent.com/368842/48972575-64cab180-f02d-11e8-9bdf-82f9008ac090.gif)

I am using the movie example schema to demonstrate here. All I did is adding this field to the "movie" document:

```
    {
      name: "movies",
      title: "Related movies",
      type: "array",
      of: [{ type: "movie" }]
    },

```

**Comment by kmelve** [comment]
Yeah, that's an irritating one! Fortunately, there's a fix for it just around the corner: https://github.com/sanity-io/sanity/pull/1060
